packages/template-ui/src/store/index.js

```diff
@@ -71,7 +71,8 @@ const initialState = {
   hasDarkHeader: false,
   hasSectionsSearch: true,
   hasCarousel: true,
-  hasFilters: true,
+  // FIXME reenable the filters and migrate them to the API:
+  hasFilters: false,
```

Comment:
Maybe we can keep some kind of filtering in the future, but just at leaf level, not with content inside folders.

Reply:
I think we should do everything through APIs.